### PR TITLE
tests: BenchmarkCreate: cover empty context scenario

### PR DESCRIPTION
**Pull request by jgehrcke**
Tiny test coverage extension, motivated by https://github.com/conbench/conbench/issues/365.

### CODE REVIEW

**Comment by jonkeane** [comment]
This shouldn't block this PR itself, but any idea what's up with the coverage allegedly decreasing (and only for machine_info?) here? I can imagine adding a test like this might not increase line coverage — but decreasing makes me wonder if something is off with our coverage setup?

**Comment by austin3dickey** [comment]
I've noticed that Coveralls has been saying `conbench/machine_info.py`'s coverage has been decreasing on every PR for a long time. It also doesn't display the source when you click through the links, which (IMO) makes it a much less useful tool. I'll create an issue.

File: conbench/tests/api/test_benchmarks.py [hunk 1]
```diff
@@ -843,3 +843,50 @@ def test_neither_stats_nor_error_field_is_present(self, client):
...
+        self.authenticate(client)
+        payload = _fixtures.VALID_PAYLOAD.copy()
+        payload["run_id"] = _uuid()
+        payload["batch_id"] = _uuid()
+
+        payload["context"] = {}
+        resp = client.post(self.url, json=payload)
+        assert resp.status_code == 201, f"unexpected response: {resp.text}"
+
+        bechmark_id = resp.json["id"]
```
Comment (austin3dickey) [inline conbench/tests/api/test_benchmarks.py]: nit:
```suggestion
        benchmark_id = resp.json["id"]
```
(and down below when you use it again)

**Review by austin3dickey — APPROVED** [review]
Thanks, looks great!

File: conbench/tests/api/test_benchmarks.py (hunk 1 above)
Comment (jgehrcke) [reply conbench/tests/api/test_benchmarks.py]: Thanks!

**Comment by jgehrcke** [comment]
Going ahead with this -- thanks for feedback!

**Pull request merged**